src/foundry/client/apps/forms/effect-config.d.ts

```diff
@@ -24,21 +23,13 @@ declare global {
      * });
      * ```
      */
-    static get defaultOptions(): ActiveEffectConfig.Options;
+    static override get defaultOptions(): ActiveEffectConfig.Options;
 
-    /** @override */
-    get title(): string;
+    override get title(): string;
 
-    /**
-     * @override
-     *
-     * @remarks The implementation doesn't return a `Promise` but the return type includes it to allow extending
```

Comment:
The remark should stay.

Reply:
I also removed this intentionally, because by now, this is a pattern we use all over the place for Application classes, and we usually don’t add the remark. But if you prefer it to stay, I can put it back.